examples/calculations/submit_singlepoint.py

```diff
@@ -0,0 +1,167 @@
+"""Example code for submitting single point calculation"""
+
+from pathlib import Path
+from typing import Union
+
+from ase.build import bulk
+from ase.io import read
+import click
+
+from aiida.common import NotExistent
+from aiida.engine import run_get_node
+from aiida.orm import Str, StructureData, load_code, load_node
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.data.model import ModelData
+
+
+def load_model(string: Union[str, None], architecture: str) -> ModelData:
+    """
+    Load a model from a given string.
+
+    If the string represents a file path, the model will be loaded from that path.
+    Otherwise, the model will be downloaded from the specified location.
+
+    Parameters
+    ----------
+    string : Union[str, None]
+        The string representing either a file path or a URL for downloading the model.
+    architecture : str
+        The architecture of the model.
+
+    Returns
+    -------
+    ModelData or None
+        The loaded model if successful, otherwise None.
+    """
+    if string is None:
+        model = None
+    elif (file_path := Path(string)).is_file():
+        model = ModelData.local_file(file_path, architecture=architecture)
+    else:
+        model = ModelData.download(string, architecture=architecture)
+    return model
+
+
+def load_structure(struct: Union[str, Path, int, None]) -> StructureData:
```

Comment:
This might be a useful place for [`functools.singledispatch`](https://docs.python.org/3/library/functools.html#functools.singledispatch), but then again, I always think that.

Reply:
this is an example let us keep it simple for the normal user...